host/4/bullseye/amd64/python/python311/python311.Dockerfile

```diff
@@ -0,0 +1,101 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+
+# Build the runtime from source
+ARG HOST_VERSION=4.17.1
```

Comment:
Make sure host version is equal on all images. We are on 4.17.3 rn

Reply:
Updated to latest. thanks